vocode/streaming/transcriber/base_transcriber.py

```diff
@@ -58,19 +58,18 @@ def terminate(self):
         pass
 
 
-class BaseAsyncTranscriber(AbstractTranscriber[TranscriberConfigType], AsyncWorker):
+class BaseAsyncTranscriber(AbstractTranscriber[TranscriberConfigType], AsyncWorker[bytes]):  # type: ignore
```

Comment:
will get fixed in vocode 0.1.0 (fix is already up in the `vocode-core-0.1.0` branch